ospo-book/content/en/06-chapter.md

```diff
@@ -1,7 +1,103 @@
 
-# TBD
+## Sustainability of Stewards in OSS
+
+Content has been curated from the [OSPOlogyLive Apeldoorn Roundtable Discussion](https://github.com/todogroup/ospology/discussions/438#discussioncomment-8619964) and based on the CRA Definition of Steward.
+
+### Government Funding Challenges
+`🏛 Social-Gov-Oriented`
+
+An ongoing challenge in governments is the inherent difficulty governments face in allocating funds for resources that are available for free. There is a strategic interest from governments to support open source software, but hurdles exist, including the potential complications arising from direct financial requests. This underscores the need for exploring alternative support infrastructures for stewardship organizations without directly soliciting government funding.
+
+### The Commons Dilemma
+`🏛 Social-Gov-Oriented`
+
+The tragedy of the commons is notably present in the open source world, where disproportionate consumption by certain users exacerbates sustainability issues. Discussions suggested the need to strengthen infrastructure support to mitigate these disparities and ensure a more balanced contribution and consumption model within the open source  community.
+
+### Community Norms and Funding Strategies
+`🏛 Social-Gov-Oriented`
+
+Establishing community norms around the costs associated with SBOM delivery and compliance has been proposed as an idea. This could involve suggesting a donation model based on organization size to account for the rising costs of compliance. Additionally, the role of risk mitigation as a motivation for organizations to invest in open source was discussed, alongside the potential for government agencies to contribute indirectly to open source projects through third parties.
+
+### Policy and Funding Initiatives
+`🏛 Social-Gov-Oriented`
+
+The possibility of the EU mandating open source as a requirement for Horizon funding suggests a growing recognition of the value of open source at the policy level. There is an ongoing discussion on strategic funding models, like the German sovereign tech fund, and the potential for government roles in supporting open source.
+
+### Challenges in open source Sustainability
+`📈 Business-Oriented`
+
+The underfunding of crucial projects like Django, and the broader issue of maintainers being underpaid or volunteering, underscores the sustainability challenges within the open source ecosystem. The critical role of vulnerability management and the need for corporate contributions to open source have been highlighted as areas requiring attention and action.
+
+### Training and Education as Funding Avenues
+`🏛 Social-Gov-Oriented`
+
+The potential for integrating training about open source within government contracts is seen as a means to provide both funding for open source organizations and valuable training for employees. This represents an approach to leveraging existing procurement processes to support open source sustainability.
```

Comment:
Can we be clearer about how trainings contribute to funding open-source organizations? Is it expected that the organization would sell training as a service?